tests/unit/core_tests.py

```diff
@@ -14,11 +15,21 @@
     Span,
     SpanObserver,
     TraceInfo,
+    AuthenticationContext,
+    UndefinedSecretsException,
+    UndefinedAuthenticationError,
 )
 from baseplate.integration import WrappedRequestContext
+from baseplate.file_watcher import FileWatcher
+from baseplate.secrets import store
 
 from .. import mock
 
+cryptography_installed = True
+try:
+    import cryptography
+except:
+    cryptography_installed = False
```

Comment:
This is ugly and I don't like it very much, but that fact may be a feature rather than a bug and will incentivize me to clean this up sooner rather than later.